### PR TITLE
Enable Audio Block

### PR DESCRIPTION
**Pull request by ceyhun**
See Gutenberg PR for changes and testing: https://github.com/WordPress/gutenberg/pull/28594

PR submission checklist:

- [x] I have considered adding unit tests where possible (in the future).
- [x] I have considered if this change warrants user-facing release notes [more info](https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/Release-notes.md) and have added them to [RELEASE-NOTES.txt](https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/RELEASE-NOTES.txt) if necessary.

### CODE REVIEW

**Comment by fluiddot** [comment]
👋  @ceyhun , we're cutting the release now, however since this PR will be included in the release I'll leave it in the current milestone. Once it's merged please let me know and I'll cherry pick the changes in the release branch, thanks!

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 Unreleased
 ------
 * [**] Add proper handling for single use blocks such as the more block [#3042]
+* [***] New Block: Audio
```
Comment (jd-alexander) [inline RELEASE-NOTES.txt:3]: @ceyhun Based on the other release note changes in this file the three-dot release note items seem to go above the two-dot ones. I am not sure if this has to be changed. LMK.

Comment (ceyhun) [reply RELEASE-NOTES.txt:3]: Will update it 👍

**Comment by ceyhun** [comment]
Gutenberg side changes are now included after https://github.com/wordpress-mobile/gutenberg-mobile/pull/3079 is merged to `develop`, this PR now only updates the release notes.

**Review by jd-alexander — APPROVED** [review]
Thanks for these changes @ceyhun  LGTM 🚢

**Pull request merged**